app/presenters/quadicons/quadrants/auth_status.rb

```diff
@@ -0,0 +1,21 @@
+module Quadicons
+  module Quadrants
+    class AuthStatus < Quadrants::Base
+      def path
+        "100/#{h(img)}.png"
```

Comment:
Please don't do this, it's really painful to find the images when the string is without `100/` and `.png`. Thanks :)